trek.js

```diff
@@ -0,0 +1,81 @@
+const baseURL = 'https://trektravel.herokuapp.com/trips';
+
+$(document).ready(() => {
+  $('#reservationForm').hide();
+  $('#singleTripDetails').hide();
+  $('#displayArea').hide();
```

Comment:
Instead of saying `.hide()` in `$(document).ready`, you can also set these to `display: none` in your CSS (this is what `.hide()` does behind the scenes). The only benefit is that this prevents them from flashing on the page for a moment when it first loads.